<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>586</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>889</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>9999</width>
    <height>9999</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>超市商品信息管理系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/Pictures/Saved Pictures/wxyd.png</normaloff>:/Pictures/Saved Pictures/wxyd.png</iconset>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgba(42, 58, 67, 240);</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <widget class="QTreeWidget" name="treeWidget">
      <property name="minimumSize">
       <size>
        <width>190</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>190</width>
        <height>9999</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(166, 199, 219, 150);
color: rgba(42, 58, 67, 240);
font: 11pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>9999</width>
        <height>60</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(42, 58, 67, 240);</string>
      </property>
      <layout class="QGridLayout" name="gridLayout" columnstretch="4,5,4,0,0,0,0">
       <item row="0" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>597</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>超市商品信息管理系统</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QPushButton" name="bin_exit">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::DefaultContextMenu</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(197, 225, 245, 150);
color: rgb(35, 42, 48);
hover:{background-color: rgba(55, 62, 76, 150);}
</string>
         </property>
         <property name="text">
          <string>关闭</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" colspan="4">
        <widget class="QWidget" name="widget_2" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>10</y>
            <width>30</width>
            <height>30</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/user.jpg);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="lb_username">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>0</y>
            <width>161</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>用户名：admin</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="maximumSize">
       <size>
        <width>9999</width>
        <height>9999</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(179, 198, 214, 100);
QStackedWidge QToolButton {background-color: rgb(34, 44, 51);}
color: rgb(220, 238, 255);</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="5" column="0">
         <widget class="QWidget" name="widget_4" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777214</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QPushButton" name="bin_simulation">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>25</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(205, 216, 239);</string>
             </property>
             <property name="text">
              <string>模拟数据</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="bin_clear">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>25</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(205, 216, 239);</string>
             </property>
             <property name="text">
              <string>清空</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>461</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="lb_cnt">
             <property name="styleSheet">
              <string notr="true">color: rgb(9, 29, 14);</string>
             </property>
             <property name="text">
              <string>商品数量：</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QWidget" name="widget_3" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,3,3,0,9,0,3">
           <item>
            <widget class="QPushButton" name="bin_add">
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(12, 14, 19);
background-color: rgb(190, 205, 224);</string>
             </property>
             <property name="text">
              <string>增加</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bin_update">
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(12, 14, 19);
background-color: rgb(190, 205, 224);</string>
             </property>
             <property name="text">
              <string>修改</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bin_delete">
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(12, 14, 19);
background-color: rgb(190, 205, 224);</string>
             </property>
             <property name="text">
              <string>删除</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>28</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLineEdit" name="le_search">
             <property name="toolTip">
              <string>可查询名称</string>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgba(42, 58, 67, 240);
background-color: rgb(166, 199, 219);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>19</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="bin_search">
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(12, 14, 19);
background-color: rgb(190, 205, 224);</string>
             </property>
             <property name="text">
              <string>搜索</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QTableWidget" name="tableWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>9999</width>
            <height>9999</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QTableView::item:hover
{
	background-color: rgba(47, 70, 74, 50);
}

QTableView::item:selected
{
	background-color: rgba(103, 141, 202, 150);
}

QTableWidget::item {
	background-color: rgb(192, 217, 231)
}
QHeaderView::section {
    color: white;
    font-size: 12px; /* 设置字体大小为12像素 */
	font-widget: bold;
	padding-top: 5px;
	height: 20px;
	background-color: rgba(42, 58, 67, 240);
	border:0.2px solid rgb(67, 93, 101);
}
QTableWidget {
    border: 1px solid rgb(67, 93, 101);; /* 设置表格外边框粗细为1像素 */
}

QTableWidget::item {
    color: black; /* 将所有表格内字体颜色设置为黑色 */
}

QTableWidget::item[row='0'] {
    color: white; /* 将第一行表头的字体颜色恢复为默认值 */
}
</string>
          </property>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>25</number>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>105</number>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>25</number>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>30</number>
          </attribute>
          <row/>
          <column>
           <property name="text">
            <string>新建列</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>889</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
